mozdownload/scraper.py

```diff
@@ -686,12 +688,15 @@ def binary_regex(self):
         """Return the regex for the binary."""
         regex = {'linux': r'^%(BINARY_NAME)s-%(VERSION)s\.%(EXT)s$',
                  'linux64': r'^%(BINARY_NAME)s-%(VERSION)s\.%(EXT)s$',
+                 'linux-arm64': r'^%(BINARY_NAME)s-%(VERSION)s\.%(EXT)s$',
```

Comment:
Good catch. Given that we don't have tests in eg. `tests/release_scraper/test_release_scraper.py` we didn't detect that we actually had this bug. 

Would you mind adding a test for arm64 builds to each of the scrapers? Thanks!